### PR TITLE
UI fix cotabes phone#180

### PR DESCRIPTION
**Pull request by lovelgeorge99**

### CODE REVIEW

File: src/app/categories/components/CategoryProjectRankingCardWithMetrics.tsx [hunk 1]
```diff
@@ -75,7 +75,7 @@ const CategoryProjectRankingCardWithMetrics = ({
 			<div className='w-full select-none rounded-2xl pb-5'>
 				<div className='relative mb-4'>
 					{project.image ? (
-						<div className='mx-auto h-[360px] w-[360px]'>
+						<div className='xxs:w-[250px] xxs:h-[250px] xs:h-[360px] xs:w-[360px] mx-auto '>
```
Comment (mmahdigh) [inline src/app/categories/components/CategoryProjectRankingCardWithMetrics.tsx:78]: We instead do: `w-full h-full w-max[360px] h-max-[360px] mx-auto `. Wdyt?

**Comment by mmahdigh** [comment]
There's also another scenario for small phones where it's still `flex-row` however the progress bar and the metric table are not centered. It'd be fantastic if you could take a look at this too:

image

File: src/app/categories/components/CategoryProjectRankingCardWithMetrics.tsx (hunk 1 above)
Comment (lovelgeorge99) [reply src/app/categories/components/CategoryProjectRankingCardWithMetrics.tsx:78]: yes this is better..thanks

**Comment by lovelgeorge99** [comment]
image

**Pull request merged**